src/cascadia/TerminalConnection/AzureConnection.cpp

```diff
@@ -115,6 +117,8 @@ namespace winrt::Microsoft::Terminal::TerminalConnection::implementation
     // - creates the output thread (where we will do the authentication and actually connect to Azure)
     void AzureConnection::Start()
     {
+        _httpClient = winrt::Windows::Web::Http::HttpClient{};
+        _httpClient.DefaultRequestHeaders().UserAgent().TryParseAdd(HttpUserAgent);

```

Comment:
What happens if you leave this empty? Does it get a default user-agent?

Reply:
Testing

Reply:
there is NO user agent